### PR TITLE
Ember Tooltip

### PR DESCRIPTION
**Pull request by MelSumner**
This adds the ember-tooltip addon into the demo app.

### CODE REVIEW

File: tests/unit/routes/addons/ember-tooltips-test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('route:addons/ember-tooltips', 'Unit | Route | addons/ember tooltips', {
+  // Specify the other units that are required for this test.
+  // needs: ['controller:foo']
+});
+
+test('it exists', function(assert) {
```
Comment (ghost) [inline tests/unit/routes/addons/ember-tooltips-test.js:8]: Shouldn't be adding tests without filling them out. Can we delete theses before the merge?

File: app/templates/addons/ember-tooltips.hbs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<h3 class="section-title"><a href="https://github.com/sir-dunxalot/ember-tooltips" rel="external">Ember Tooltips</a></h3>
+<div class="example">
+	<div class="output">
+	{{#demo-text tooltipContent='This is the tooltip'}}
+  	Hover over me!
+	{{/demo-text}}
+
+	</div>
+	<div class="source">
+		<pre><code>\{{#demo-text tooltipContent='This is the tooltip'}}
```
Comment (ghost) [inline app/templates/addons/ember-tooltips.hbs:10]: Might be worth also including an example of the tooltip in component example

https://github.com/sir-dunxalot/ember-tooltips#using-as-a-component

File: app/styles/app.scss [hunk 3]
```diff
@@ -218,7 +218,7 @@ article.aria-roles {
     margin-bottom: 0.5em;
   }
 .demo-div {
-  flex: 1 1 1;
```
Comment (ghost) [inline app/styles/app.scss:221]: we might want to consider using something other than flexbox for browser support.

File: tests/unit/routes/addons/ember-tooltips-test.js (hunk 1 above)
Comment (MelSumner) [inline tests/unit/routes/addons/ember-tooltips-test.js:8]: I'll get some tests in here today. Good point.

File: app/templates/addons/ember-tooltips.hbs (hunk 2 above)
Comment (MelSumner) [inline app/templates/addons/ember-tooltips.hbs:10]: sounds good.  I'll add that.

File: app/styles/app.scss (hunk 3 above)
Comment (MelSumner) [inline app/styles/app.scss:221]: Do we really anticipate traffic on IE9 or below? I'll make sure there is fallback support.

Comment (ghost) [inline app/styles/app.scss:221]: Wow flexbox support is so much further along than I thought.

I personally wouldn't worry about IE9 as long as it doesn't pose problems for AT user agents.

image

**Pull request merged**